export function caughtIn4k(error: Error): void {
  console.error(error);
}
